src/scripts/lib/server/pipeline/index.js

```diff
@@ -15,7 +15,7 @@ const flatten = function(array) {
 
 module.exports = class Pipeline {
 
-  static DATA_DIR = 'data'
+  static DATA_DIR = '@@clientData'
```

Comment:
Well I would argue that replacing this at build time is a more elegant solution. Less strain during server launch even if it's negligible. Also good excuse to leave it as it is :stuck_out_tongue_winking_eye: